tsconfig.json

```diff
@@ -1,20 +1,27 @@
 {
+    "include": ["src", "types"],
     "compilerOptions": {
-        "module": "UMD",
         "allowJs": true,
-        "checkJs": true,
-        "target": "es6",
-        "moduleResolution": "Node",
+        "module": "esnext",
+        "lib": ["dom", "esnext"],
+        "importHelpers": true,
+        "declaration": true,
+        "sourceMap": true,
+        "rootDir": "src",
+        "strict": false,
```

Comment:
Hi @vishnugopal Did you consider enabling `strict` first, perhaps independent of this PR? This is where the biggest value is.

Reply:
Yup I did, there's too many errors to fix quickly with `strict: true`. I'll definitely tackle them in a future PR. It's easier to find these now once I get the infrastructure rolling.